WordPress/src/main/res/values/site_creation_intents.xml

```diff
@@ -1,33 +1,104 @@
+        <item>@string/site_creation_intent_interior_design</item>
+        <item>@string/site_creation_intent_lifestyle</item>
+        <item>@string/site_creation_intent_local_services</item>
+        <item>@string/site_creation_intent_music</item>
+        <item>@string/site_creation_intent_news</item>
+        <item>@string/site_creation_intent_parenting</item>
+        <item>@string/site_creation_intent_people</item>
+        <item>@string/site_creation_intent_personal</item>
+        <item>@string/site_creation_intent_photography</item>
+        <item>@string/site_creation_intent_politics</item>
+        <item>@string/site_creation_intent_sports</item>
+        <item>@string/site_creation_intent_technology</item>
+        <item>@string/site_creation_intent_travel</item>
+        <item>@string/site_creation_intent_writing_poetry</item>
     </string-array>
     <string-array name="site_creation_intents_emojis">
-        <item>&#128188;</item><!-- Briefcase -->
-        <item>&#127828;</item><!-- Cheeseburger -->
-        <item>&#10084;</item><!-- Heart -->
-        <item>&#127968;</item><!-- House -->
-        <item>&#8505;</item><!-- Information -->
-        <item>&#128478;</item><!-- Newspaper -->
-        <item>&#128247;</item><!-- Camera -->
-        <item>&#9992;</item><!-- Airplane -->
+        <item>&#127912;</item><!-- 🎨 Art -->
+        <item>&#128663;</item><!-- 🚗 Automotive -->
+        <item>&#128133;</item><!-- 💅 Beauty -->
+        <item>&#128218;</item><!-- 📚 Books -->
+        <item>&#128188;</item><!-- 💼 Business -->
+        <item>&#129309;</item><!-- 🤝 Community & Non-Profit -->
+        <item>&#127979;</item><!-- 🏫 Education -->
+        <item>&#128296;</item><!-- 🔨 DIY -->
+        <item>&#128096;</item><!-- 👠 Fashion -->
+        <item>&#128176;</item><!-- 💰 Finance -->
+        <item>&#127909;</item><!-- 🎥 Film & Television -->
+        <item>&#128170;</item><!-- 💪 Fitness & Exercise -->
+        <item>&#127828;</item><!-- 🍔 Food -->
+        <item>&#127918;</item><!-- 🎮 Gaming -->
+        <item>&#10084;</item><!-- ❤️ Health -->
+        <item>&#128715;</item><!-- 🛋️ Interior Design -->
+        <item>&#127968;</item><!-- 🏠 Lifestyle -->
+        <item>&#128205;</item><!-- 📍Local Services -->
+        <item>&#127925;</item><!-- 🎵 Music -->
+        <item>&#128478;</item><!-- 🗞️ News -->
+        <item>&#128118;</item><!-- 👶 Parenting -->
+        <item>&#129489;&#8205;&#129309;&#8205;&#129489;</item><!-- 🧑‍🤝‍🧑 People -->
+        <item>&#9997;&#65039;</item><!-- ✍️ Personal -->
+        <item>&#128247;</item><!-- 📷 Photography -->
+        <item>&#128499;</item><!-- 🗳️ Politics -->
+        <item>&#9917;</item><!-- ⚽ Sports -->
+        <item>&#128187;</item><!-- 💻 Technology -->
+        <item>&#9992;</item><!-- ✈️ Travel -->
+        <item>&#128211;</item><!-- 📓 Writing & Poetry -->
+    </string-array>
+    <string-array name="site_creation_intents_defaults">
```

Comment:
I like this solution, it allows us to easily change the defaults with low impact 🙌